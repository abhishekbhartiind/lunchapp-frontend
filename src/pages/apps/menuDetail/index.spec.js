import React from 'react'
import renderer from 'react-test-renderer'
import { MockedProvider } from 'react-apollo/test-utils'
import { BrowserRouter, withRouter } from 'react-router-dom'
import MenuDetail from './index'

// eslint-disable-next-line no-undef
it('menu detail correctly', () => {
	const MenuDetails = withRouter(MenuDetail)
	const menu = renderer
		.create(
			<MockedProvider mocks={[]}>
				<BrowserRouter>
					<MenuDetails />
				</BrowserRouter>
			</MockedProvider>
		)
		.toJSON()
	// eslint-disable-next-line no-undef
	expect(menu).toMatchSnapshot()
})
