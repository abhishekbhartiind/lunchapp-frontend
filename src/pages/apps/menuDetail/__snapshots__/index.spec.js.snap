// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`menu detail correctly 1`] = `
<div
  className="ant-skeleton ant-skeleton-active"
>
  <div
    className="ant-skeleton-content"
  >
    <h3
      className="ant-skeleton-title"
      style={
        Object {
          "width": "38%",
        }
      }
    />
    <ul
      className="ant-skeleton-paragraph"
    >
      <li
        style={
          Object {
            "width": undefined,
          }
        }
      />
      <li
        style={
          Object {
            "width": undefined,
          }
        }
      />
      <li
        style={
          Object {
            "width": "61%",
          }
        }
      />
    </ul>
  </div>
</div>
`;
